spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -2173,27 +2214,27 @@ will be available under `/system`.
 
 
 [[howto-switch-off-spring-boot-security-configuration]]
-=== Switch off the Spring Boot security configuration
-If you define a `@Configuration` with `@EnableWebSecurity` anywhere in your application
-it will switch off the default webapp security settings in Spring Boot (but leave the
+=== Switch off the Spring Boot Security Configuration
+If you define a `@Configuration` with `@EnableWebSecurity` anywhere in your application,
+it switches off the default webapp security settings in Spring Boot (but leaves the
 Actuator's security enabled). To tweak the defaults try setting properties in
 `+security.*+` (see
 {sc-spring-boot-autoconfigure}/security/SecurityProperties.{sc-ext}[`SecurityProperties`]
-for details of available settings) and `SECURITY` section of
-<<common-application-properties-security,Common application properties>>.
+for details of available settings) and the `SECURITY` section of
+"`<<common-application-properties-security,Common Ppplication Properties>>`".
```

Comment:
"Ppplication" -> "Application"

Reply:
I'll just go crawl under this rock over here....

Thanks.